src/bom/extract.rs

```diff
@@ -63,37 +65,16 @@ pub fn do_bitcode_extraction(extract_options : &ExtractOptions,
     // Thus the more robust solution is to specify the output file in a temporary
     // directory, and there is already a convenient temporary directory created
     // above to hold the output llvm bitcode tar file.
-    let mut objcopy_args = Vec::new();
-    objcopy_args.push(OsString::from("--dump-section"));
-    objcopy_args.push(OsString::from(format!("{}={}", ELF_SECTION_NAME, ok_tar_name)));
-
-    objcopy_args.push(OsString::from(&extract_options.input));
-
-    let mut objres = PathBuf::new();
-    objres.push(tmp_path);
-    objres.push("discard{output-file}");
-    objcopy_args.push(OsString::from(objres));
+    let mut dummy_output = PathBuf::from(tmp_path);
+    dummy_output.push("discard{output-file}");
```

Comment:
Is this name a special directive to the library? Or is it just an arbitrary name?

Reply:
This is entirely arbitrary (and conversely, there are no "special strings" used by the chainsop library).  I've added a comment to this effect.